pages/blogs/add.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div v-if="!preview">
+    <v-row>
+      <v-col cols="6" class="mx-auto">
+        <v-text-field
+          v-model="title"
+          label="Enter Title of your blog"
+        ></v-text-field>
+        <v-text-field v-model="subtitle" label="Enter Subtitle"></v-text-field>
+        <v-text-field v-model="link" label="Enter Image Link"></v-text-field>
+        <v-textarea
+          v-model="content"
+          name="input-7-1"
+          label="Enter Blog"
+        ></v-textarea>
+        <v-btn v-if="team.length" text @click="setUser()">Next</v-btn>
+      </v-col>
+    </v-row>
+  </div>
+  <div v-else>
+    <v-row justify="center" dense>
+      <v-col cols="6">
+        <h1 class="font-weight-medium title-font">{{ title }}</h1>
+        <p class="subtitle-font">{{ subtitle }}</p>
+        <p class="font-weight-normal">{{ user.name }}</p>
+        <v-img :src="link" />
+        <div class="content-font">
+          <p>{{ content }}</p>
+        </div>
+        <br />
+        <nuxt-link to="/blogs" style="text-decoration: none">
+          <v-btn text @click="writeBlog()">Proceed</v-btn>
+        </nuxt-link>
+        <v-btn text @click="preview = false">Make Changes</v-btn>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script>
+import { db } from '@/plugins/firebase'
+import { mapState, mapActions } from 'vuex'
+export default {
+  data() {
+    return {
+      value: false,
+      title: '',
+      subtitle: '',
+      content: '',
+      count: 0,
+      preview: false,
+      link: '',
+      currentUser: {}
```

Comment:
No need to store as data, this is a read only attribute. Data is used for reactive read-write attributes